test/checker/lint_test.go

```diff
@@ -21,11 +21,13 @@ const (
 	unpopularPkg   = "unpopular"
 	nonexistentPkg = "nonexistent"
 	normalPkg      = "normal"
+	unpopularRepo  = "user/unpopularRepo"
 )
 
-// fakes the npm api for testing purposes
-func fakeNpmHandlerLint(w http.ResponseWriter, r *http.Request) {
+// fakes the npm api and GitHub api for testing purposes
+func fakeNpmGitHubHandlerLint(w http.ResponseWriter, r *http.Request) {
 	switch r.URL.Path {
```

Comment:
to make it clearer between github and npm we could concatenate the Host header and the path

Reply:
Good idea, done!